test/e2e_federation/util.go

```diff
@@ -272,6 +268,11 @@ func createService(clientset *fedclientset.Clientset, namespace, name string) (*
 	}
 	By(fmt.Sprintf("Creating federated service %q in namespace %q", name, namespace))
 
+	// Tests can be run in parallel, so we need a different nodePort for
+	// each test. [30000, 32768) is the default nodeport range and our
+	// tests just use the defaults.
+	nodePort := int32(rand.IntnRange(30000, 32768))
```

Comment:
I'd suggest....
nodePort := int32(rand.IntnRange(FederatedSvcNodePortFirst, FederatedSvcNodePortLast+1))
That way docs can be kept at the top with the other caveats.